examples/channels.rs

```diff
@@ -112,7 +112,7 @@ fn server_setup(mut net: ResMut<NetworkResource>) {
 fn client_setup(commands: &mut Commands, mut net: ResMut<NetworkResource>) {
     let mut camera = Camera2dBundle::default();
     camera.orthographic_projection.window_origin = WindowOrigin::BottomLeft;
-    commands.spawn(camera);
+    commands.spawn().insert(camera);
```

Comment:
Is this the recommended way of spawning camera in Bevy 0.5?

Reply:
It's the new way to spawn in general.

Reply:
Examples suggest `commands.spawn_bundle(OrthographicCameraBundle::new_2d());` - https://github.com/bevyengine/bevy/blob/v0.5.0/examples/2d/sprite.rs#L16

Reply:
`spawn().insert()` or `spawn_bundle()`, both work on new Bevy 0.5 engine, but seem like we need to do a rework on channel example if we decide to upgrade with 0.5 version.

Some bundle is removed or changed (Camera2Bundle)  from API.

Reply:
Since the camera bundle comprises the whole entity, it makes sense to just use `spawn_bundle`.

Reply:
Agree, I updated it.